test/cql-pytest/cassandra_tests/validation/operations/alter_test.py

```diff
@@ -69,7 +69,6 @@ def testDropListAndAddMapWithSameName(cql, test_keyspace):
         execute(cql, table, "ALTER TABLE %s DROP myCollection")
         assert_invalid(cql, table, "ALTER TABLE %s ADD myCollection map<int, int>")
 
-@pytest.mark.xfail(reason="Issue #9929")
```

Comment:
Please add "Refs #9929" in the commit message, to make this link. You can also add a few words that the previous patch added the missing support for ALTER TABLE ... DROP ... USING TIMESTAMP, which is why these tests no longer fail.

There's a fourth test in the same file, testCreateAlterKeyspaces() which also lists #9929 on its xfail. The test should still fail because of the still open #9935, but it still should claim it xfails because of an issue you are closing.

Finally, please consider adding a comment line like `# reproduces #9929` in front of tests that you modified - this comment can be useful in the future if we see this test failing on some release of Scylla and wonder what we forgot to backport, or what fix got regressed.

Reply:
> Finally, please consider adding a comment line like # reproduces #9929 in front of tests that you modified - this comment can be useful in the future if we see this test failing on some release of Scylla and wonder what we forgot to backport, or what fix got regressed.

Do you mean to add both `# Refs #9929` and `# Reproduces #9929`? For now only added refs comments.

Reply:
I asked to put "Refs #9929" in the **commit message**, not in the comments :-) This has magic meaning to github, which will later allow finding this patch when you look at issue 9929.
In the comments, I prefer to use actual English sentences, not code words like "Refs". For example, "Reproduces #9929". I think we should have both.

Reply:
@Jadw1 can this comment be resolved?